defaults/main.yml

```diff
@@ -120,13 +120,20 @@ atom_template_propel_ini: "atom/config/propel.ini"
 
 atom_template_app_yml: "atom/apps/qubit/config/app.yml"
 atom_app_upload_limit: "-1"
+
+## app.yml cache settings (prefix must be different for each site in a multisite deploy)
 atom_app_cache_engine: "sfMemcacheCache"
 atom_app_cache_engine_options:
   storeCacheInfo: "yes"
-  prefix: "atom"
+  prefix: "{{ atom_path | basename }}"
```

Comment:
This may still be an issue if Memcached is in a different server and there are multiple AtoM instances connected to it, sharing the same path in different machines. Similar to the `worker_key` setting ...

https://github.com/artefactual/atom/blob/qa/2.x/config/app.yml#L43-L50